maxim/logger/writer.py

```
@@ -339,7 +339,7 @@ def write_to_file(self, logs):
                     file.write(log.serialize() + "\n")
             return filepath
         except Exception as e:
-            scribe().info(
+            scribe().warning(
                 f"[MaximSDK] Failed to write logs to file. We will keep it in memory. Error: {e}"
```

Ruff (0.11.9)

Trailing comma missing

Add trailing comma

(COM812)